test/functional/integreatly_test.go

```diff
@@ -47,4 +47,13 @@ func TestIntegreatly(t *testing.T) {
 
 		common.RunTestCases(common.DESTRUCTIVE_TESTS, t, config)
 	})
+
+	t.Run("API Managed Multi-AZ Tests", func(t *testing.T) {
+		// Do not execute these tests unless MULTIAZ is set to true
+		if os.Getenv("MULTIAZ") != "true" {
+			t.Skip("Skipping Multi-AZ tests as MULTIAZ env var is not set to true")
+		}
+
+		common.RunTestCases(MULTIAZ_TESTS, t, config)
+	})
```

Comment:
Running these tests before destructive tests could be safer.
Or is this put here on purpose? :)

Reply:
No, it is a good idea to run these tests before the destructive ones, I'll add the change.